.gitignore

```diff
@@ -1,2 +1,5 @@
 /node_modules/
+/umd/
+/es/

```

Comment:
is it by intention? Because `index.es.js` relies on `require()` from that dir

Reply:
I'm getting this error when running the latest version of stripes-react-hotkeys:

![image](https://user-images.githubusercontent.com/640976/48848606-a0067f80-eda4-11e8-9c92-1884d15d4db4.png)

Running `stripes serve` in my `stripes-sample-platform`-folder. I'm on stripes v.1.6.1000131

Reply:
Running `yarn prepublish` in the directory worked.

Reply:
This is a yarn workspace issue: https://github.com/yarnpkg/yarn/issues/3911